### PR TITLE
Internationalization translations into Hebrew, German, Polish and Russian

### PR DESCRIPTION
**Pull request by AphroBytes**
Hebrew, German and Russian translations. Hope this can help. 

## Multilingual or copywriting? 多语言或者文案？
Please submit directly, we will evaluate.
## Dev branch!
Any changes should be submitted to the dev branch.
任何改动，请提交到 dev 分支。
--Edited to add Polish

### CODE REVIEW

**Comment by TCOTC** [comment]
Please add English for all history operations, like this:

image

**Comment by AphroBytes** [comment]
That is now done.

**Comment by TCOTC** [comment]
Good. There are still a few codes that need to be modified, please refer to: [`59bf2ae` (#12562)](https://github.com/siyuan-note/siyuan/pull/12562/commits/59bf2aeb0d8e712e2ae741045c1156889521ad71)

**Comment by AphroBytes** [comment]
That's now done, is there any others that need modifying? I didn't see anymore? Thank you

**Comment by TCOTC** [comment]
A few problems with code indentation.

Spaces should be used here:

image

image

**Comment by AphroBytes** [comment]
Apologies, should be ok now?

File: app/src/constants.ts [hunk 1]
```diff
@@ -195,6 +195,9 @@ export abstract class Constants {
         es_ES: "20210808180117-6v0mkxr",
         ja_JP: "20240530133126-axarxgx",
         it_IT: "20210808180117-6v0mkxr",
+        de_DE: "20241003122725-f93k2y7",
+        he_IL: "20241003122725-f93k2y7",
+        ru_RU: "20241003122725-f93k2y7",
```
Comment (TCOTC) [inline app/src/constants.ts:200]: The added language should point to the English user guide: "20210808180117-6v0mkxr"

**Review by TCOTC — CHANGES_REQUESTED** [review]

File: app/src/constants.ts (hunk 1 above)
Comment (AphroBytes) [reply app/src/constants.ts]: Keeping the new dates or should I use the original date?

File: app/src/constants.ts [hunk 2]
```diff
@@ -195,9 +195,9 @@ export abstract class Constants {
         es_ES: "20210808180117-6v0mkxr",
         ja_JP: "20240530133126-axarxgx",
         it_IT: "20210808180117-6v0mkxr",
-        de_DE: "20241003122725-f93k2y7",
-        he_IL: "20241003122725-f93k2y7",
-        ru_RU: "20241003122725-f93k2y7",
+        de_DE: "20241003122725-6v0mkxr",
+        he_IL: "20241003122725-6v0mkxr",
+        ru_RU: "20241003122725-6v0mkxr",
```
Comment (TCOTC) [inline app/src/constants.ts:200]: You forgot to change the previous section to "20210808180117"

**Review by TCOTC — CHANGES_REQUESTED** [review]

File: app/electron/init.html [hunk 3]
```diff
@@ -212,6 +212,10 @@ <h2>思源笔记</h2>
             <option value="zh_CHT">繁體中文</option>
             <option value="ja_JP">日本語</option>
             <option value="it_IT">Italiano</option>
+            <option value="de_DE">Deutsch</option>
+            <option value="he_IL">עברית</option>
+            <option value="ru_RU">Русский</option>
+
```
Comment (TCOTC) [inline app/electron/init.html:218]: There is an extra blank line that needs to be removed

**Review by TCOTC — CHANGES_REQUESTED** [review]

File: app/electron/init.html [hunk 4]
```diff
@@ -212,6 +212,10 @@ <h2>思源笔记</h2>
             <option value="zh_CHT">繁體中文</option>
             <option value="ja_JP">日本語</option>
             <option value="it_IT">Italiano</option>
+			<option value="de_DE">Deutsch</option>
```
Comment (TCOTC) [inline app/electron/init.html:215]: problem with code indentation

**Review by TCOTC — CHANGES_REQUESTED** [review]

File: app/electron/init.html (hunk 4 above)
Comment (AphroBytes) [reply app/electron/init.html]: Oh, thought I'd fixed that already apologies.

**Comment by TCOTC** [comment]
Spaces are also required here:

image

File: app/appearance/langs/de_DE.json [hunk 5]
```diff
@@ -0,0 +1,1558 @@
...
+  "pageCountAndHistoryCount": "Insgesamt ${x} Seiten, ${y} Historien",
+  "visitCommunityShare": "Besuchen Sie die Community-Sharing",
+  "clearContextSucc": "Der Kontext wurde gelöscht",
+  "emptyMobilePlaceholder": "Etwas aufzeichnen",
+  "emptyPlaceholder": "Etwas schreiben, ' / ' für Befehle",
+  "publish": "Veröffentlichen",
+  "publishService": "Veröffentlichungsdienst",
+  "publishServiceNotStarted": "Veröffentlichungsdienst nicht gestartet",
+  "publishServiceTip": "Wenn aktiviert, wird der Veröffentlichungsdienst gestartet. Dieser Dienst veröffentlicht den Inhalt des aktuellen Arbeitsbereichs im Nur-Lese-Modus im lokalen Netzwerk.",
+  "publishServicePort": "Diensteport",
```
Comment (Macavity) [inline app/appearance/langs/de_DE.json:17]:
```suggestion
  "publishServicePort": "Dienst-Port",
```

**Comment by 88250** [comment]
Can you please let me know if this change can be merged?

**Comment by AphroBytes** [comment]
> Can you please let me know if this change can be merged?

yes, ready to be merged now. As long as everyone agrees and nothing further needs fixing, which as far as I can tell it does not?

Thanks

**Comment by TCOTC** [comment]
Spaces should be used here:

image

There are no mistakes other than this one.

**Comment by 88250** [comment]
感谢你的贡献，思源有你更精彩！
Thank you for your contribution. SiYuan will be more wonderful with you!

**Pull request merged**